spec/dir_class_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'spec_helper'
+
+describe "Ext3::Dir class methods" do
+  def cassette_path
+    "spec/cassettes/dir_class.yml"
+  end
+
+  describe ".[]" do
+    it "should return empty array when in a nonexistent directory" do
+      VirtFS.cwd = "/not_a_dir" # bypass existence checks.
+      expect(VirtFS::VDir["*"]).to match_array([])
+    end
+
+    it "should enumerate the same file names as the standard Dir.glob - simple glob" do
```

Comment:
@movitto Maybe the confusion is in the test doesn't seem to be testing what it states:
`"should enumerate the same file names as the standard Dir.glob - simple glob"`
It's not calling the standard `Dir.glob`, so how can it be testing that they return the same file names?

In the VirtFS tests, the mounted filesystem is NativeFS, so the same files and directories are visible through VirtFS as well as through the standard Ruby interface, so we can compare the return values. For Ext3, the files and directories ore only visible through VirtFS, so you can't compare the return values with the standard call.